src/pages/RegisterRequestSent.tsx

```diff
@@ -20,44 +20,50 @@ const RegisterRequestSent = () => {
             {
                 title: translate('fullName'),
                 value: user.fullName,
+                span: 2
             },
             {
                 title: translate('id'),
-                value: user.id
+                value: user.id,
+                span: 1
             },
             {
                 title: translate('birthDate'),
                 value: user.birthDate,
+                span: 1
             },
             {
                 title: translate('email'),
                 value: user.email,
                 hasEditButton: true,
                 goTo: `/profile/edit-email`,
+                span: 1
             },
             {
                 title: translate('phone'),
                 value: user.phone,
+                span: 1
             },
             {
-                title: 'Miembros de la familia',
+                title: 'Miembros unidad familiar',
                 value: getFamilyMembers(user.id)?.length.toString() || '0',
+                span: 2
             }
         ];
     };
 
     return (
-        <div className="flex h-screen items-center justify-center p-8 bg-primary-color text-secondary-color">
+        <div className="h-screen flex flex-col justify-end bg-primary-color px-4 pb-4">
             <div className="app-logo">
                 <LogoFesbalWhiteIcon/>
             </div>
-            <div className="mt-8 flex w-full flex-col gap-4 self-end rounded-xl bg-white p-8 md:w-1/2 lg:w-1/3">
+            <div className="mx-auto w-full md:w-1/2 lg:w-1/3 px-8 pb-4 text-center bg-white rounded-xl text-secondary-color z-10">
                 <RequestSentIcon/>
-                <div className="flex flex-col items-center justify-center">
-                    <h1 className="mb-5 font-big-title">{translate("title")}</h1>
-                    <ul className="flex flex-col gap-0.5">
+                <div className="flex flex-col">
+                    <h1 className="mb-4 font-big-title">{translate("title")}</h1>
+                    <ul className="grid grid-cols-2 gap-3 pb-2 text-left justify-between">
                         {getPersonalData().map((item, index) => (
-                            <RegisterPersonalDataItem title={item.title} value={item.value} key={index}/>
+                            <RegisterPersonalDataItem title={item.title} value={item.value} span={item.span} key={index}/>
```

Comment:
I would remove the span prop. If this is related to a classname/style you would apply for different situations, I would name this prop as `classname`